main.cpp

```diff
@@ -20,6 +22,7 @@ int main()
     std::shared_ptr<glfw_window> window = nullptr; // Should be declared here to destroy in the last turn
 
     // Create instance of render_context.
+    // NOTE: each instance of OEP should have its own instance of context
```

Comment:
// NOTE: each instance ort (offscreen render target)  should have its own instance of rc (render context)